src/utils/storage/index.ts

```diff
@@ -10,7 +10,7 @@ export const storage = new ImmortalStorage(stores)
 
 const PREFIX = `v2_${getNetwork()}`
 
-export const loadFromStorage = async (key) => {
+export const loadFromStorage = async (key: string): Promise<any> => {
```

Comment:
Usage of any is forbidden, I think you can use generics here, so the actual usage would be

```
const safes = loadFromStorage<SafeType>(SAFE_KEY)
```

Reply:
I will just remove the types on that file and make the fix, adding types for every result of `loadFromStorage` will be a lot of effort that should be made on another pr. We need to to dispatch this issue on the release

Reply:
How’s adding a generic for single function takes a lot of effort?

Reply:
I ask you to correctly type this function, not every call

Reply:
`loadFromStorage` is used in a lot of places with different results, I just added types for what we need

Reply:
I don’t understand what is the problem there

Reply:
You type this function using genetics, next step for us (not in this PR) would be correctly type this (passing type argument) in places where we use it